apps/api/src/app/workflows-v2/usecases/generate-preview/generate-preview.usecase.ts

```diff
@@ -17,74 +18,100 @@ import {
   Instrument,
   InstrumentUsecase,
   PinoLogger,
+  sanitizePreviewControlValues,
 } from '@novu/application-generic';
 import { captureException } from '@sentry/node';
 import { PreviewStep, PreviewStepCommand } from '../../../bridge/usecases/preview-step';
 import { FrameworkPreviousStepsOutputState } from '../../../bridge/usecases/preview-step/preview-step.command';
 import { BuildStepDataUsecase } from '../build-step-data';
 import { GeneratePreviewCommand } from './generate-preview.command';
-import { PrepareAndValidateContentUsecase } from '../validate-content/prepare-and-validate-content/prepare-and-validate-content.usecase';
 import { BuildPayloadSchemaCommand } from '../build-payload-schema/build-payload-schema.command';
 import { BuildPayloadSchema } from '../build-payload-schema/build-payload-schema.usecase';
+import {
+  extractLiquidTemplateVariables,
+  TemplateParseResult,
+  Variable,
+} from '../../util/template-parser/liquid-parser';
+import { pathsToObject } from '../../util/path-to-object';
+import { HydrateEmailSchemaUseCase } from '../../../environments-v1/usecases/output-renderers';
+import { PrepareAndValidateContentUsecase } from '../validate-content/prepare-and-validate-content/prepare-and-validate-content.usecase';
 
 const LOG_CONTEXT = 'GeneratePreviewUsecase';
 
+type DestructuredControlValues = {
+  tiptapControlValues: { emailEditor?: string | null; body?: string | null } | null;
```

Comment:
@djabarovgeorge Didn't we replace emailEditor with body? I am surprised we still use it here. Is this maybe for backward compatibility? In production we don't have any email step controls yet so it shouldn't be a concern.

Reply:
this one was before we introduced email.body, we already removed the old emailEditor